crates/node-core/src/args/network.rs

```diff
@@ -90,19 +90,24 @@ pub struct NetworkArgs {
     #[arg(long)]
     pub max_inbound_peers: Option<usize>,
 
-    /// Soft limit for the byte size of a `PooledTransactions` response on assembling a
-    /// `GetPooledTransactions` request. Spec'd at 2 MiB.
-    ///
-    /// <https://github.com/ethereum/devp2p/blob/master/caps/eth.md#protocol-messages>.
-    #[arg(long = "pooled-tx-response-soft-limit", value_name = "BYTES", default_value_t = SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESPONSE, help = "Sets the soft limit for the byte size of pooled transactions response. Specified at 2 MiB by default. This is a spec'd value that should only be set for experimental purposes on a testnet.",long_help = None)]
+    /// Experimental, for usage in research. Sets the max accumulated byte size of a transactions
+    /// to pack in one responses.
```

Comment:
```suggestion
    /// Experimental, for usage in research. Sets the max accumulated byte size of a transaction
    /// to pack in one response.
```

unless its meant to be

```suggestion
    /// Experimental, for usage in research. Sets the max accumulated byte size of transactions
    /// to pack in one response.
```